components/ActionBar/index.js

```diff
@@ -144,7 +143,10 @@ const ActionBar = ({
   const displayMinutes = readingMinutes % 60
   const displayHours = Math.floor(readingMinutes / 60)
   const forceShortLabel =
-    mode === 'article-overlay' || mode === 'feed' || mode === 'bookmark'
+    mode === 'article-overlay' ||
+    mode === 'feed' ||
+    mode === 'bookmark' ||
+    mode === 'seriesEpisode'
```

Comment:
IMO we should decide if `ActionBar` modes are camelCase or minus separated. I'd vote for camelCase and would change `article-overlay` to `articleOverlay`